docker/docker-compose.yml

```diff
@@ -95,6 +105,11 @@ services:
     depends_on:
       db: # Disable this if you are using an external Postgres database
         condition: service_healthy
+    healthcheck:
+      test: [ "CMD", "bash", "-c", "printf \\0 > /dev/tcp/localhost/3000" ]
```

Comment:
This wouldn't work because PostgREST does not expose any shell.

```bash
docker exec -it rest bash
OCI runtime exec failed: exec failed: unable to start container process: exec: "bash": executable file not found in $PATH: unknown
```

Reply:
Actually I don't understand, I have a shell with PostgREST.
```sh
$ docker compose ps
WARN[0000] The "MFA_ENABLED" variable is not set. Defaulting to a blank string. 
NAME                             IMAGE                                 COMMAND                  SERVICE             CREATED             STATUS                  PORTS
realtime-dev.supabase-realtime   supabase/realtime:v2.1.0              "/usr/bin/tini -s -g…"   realtime            13 hours ago        Up 13 hours (healthy)   
supabase-auth                    supabase/gotrue:v2.40.1               "gotrue"                 auth                20 hours ago        Up 20 hours (healthy)   
supabase-db                      supabase/postgres:14.1.0.89           "docker-entrypoint.s…"   db                  41 hours ago        Up 20 hours (healthy)   0.0.0.0:5432->5432/tcp, :::5432->5432/tcp
supabase-imgproxy                darthsim/imgproxy:v3.11               "imgproxy"               imgproxy            20 hours ago        Up 20 hours (healthy)   8080/tcp
supabase-kong                    kong:2.8.1                            "/docker-entrypoint.…"   kong                41 hours ago        Up 20 hours (healthy)   0.0.0.0:8000->8000/tcp, :::8000->8000/tcp, 8001/tcp, 0.0.0.0:8443->8443/tcp, :::8443->8443/tcp, 8444/tcp
supabase-meta                    supabase/postgres-meta:v0.52.1        "docker-entrypoint.s…"   meta                41 hours ago        Up 20 hours             8080/tcp
supabase-rest                    postgrest/postgrest:v9.0.1.20220717   "/bin/sh -c postgrest"   rest                13 hours ago        Up 13 hours (healthy)   3000/tcp
supabase-storage                 supabase/storage-api:v0.26.1          "docker-entrypoint.s…"   storage             13 hours ago        Up 13 hours (healthy)   5000/tcp
supabase-studio                  supabase/studio:20221214-4eecc99      "docker-entrypoint.s…"   studio              14 hours ago        Up 14 hours (healthy)   0.0.0.0:3000->3000/tcp, :::3000->3000/tcp
```
```sh
$ docker compose exec -it rest bash
WARN[0000] The "MFA_ENABLED" variable is not set. Defaulting to a blank string. 
I have no name!@a9f8383c3e5e:/$ 
```

And also healthy seems to work well.

```sh
$ docker inspect -f '{{json .State.Health.Log }}' supabase-rest
[{"Start":"2023-01-21T06:55:47.488275093Z","End":"2023-01-21T06:55:47.525642851Z","ExitCode":0,"Output":""},{"Start":"2023-01-21T06:55:52.532195254Z","End":"2023-01-21T06:55:52.575029984Z","ExitCode":0,"Output":""},{"Start":"2023-01-21T06:55:57.585664639Z","End":"2023-01-21T06:55:57.630297652Z","ExitCode":0,"Output":""},{"Start":"2023-01-21T06:56:02.636976463Z","End":"2023-01-21T06:56:02.696038827Z","ExitCode":0,"Output":""},{"Start":"2023-01-21T06:56:07.704685445Z","End":"2023-01-21T06:56:07.753510107Z","ExitCode":0,"Output":""}]
```

Reply:
Now I see. It's because I'm on linux ARM64 and you are on linux AMD.
The images are quite different if you search them on Docker Hub.
Will remove the Healthcheck

Reply:
Sounds good! Tagging @steve-chavez to see if we can make the upstream image more consistent between architectures

Reply:
Relevant link: https://hub.docker.com/layers/postgrest/postgrest/v9.0.1.20220717/images/sha256-3914bbc153b2fe04c5648921cbf123859c2bae58e0f654a24f1c450e0ccba664?context=explore